### PR TITLE
Flesh out POM information.

### PR DESCRIPTION
**Pull request by DanAlbert**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -73,16 +73,53 @@ subprojects {
...
+                    from(components["java"])
+
+                    pom {
+                        name.set(extra["pomName"] as String)
+                        description.set(extra["pomDescription"] as String)
+                        url.set("https://google.github.io/prefab/")
+                        licenses {
+                            license {
+                                name.set(
+                                    "The Apache Software License, Version 2.0"
```
Comment (jmgao) [inline build.gradle.kts]: "The" here seems weird: maven omits it: https://maven.apache.org/pom.html#Licenses

Comment (DanAlbert) [reply build.gradle.kts]: I copied our existing POM for the gradle plugin, but Apache would certainly know best :) The actual values people put in this field are all over the place: https://github.com/google/prefab/blob/master/config/allowed_licenses.json

Comment (jmgao) [reply build.gradle.kts]: Yeah, it looks like Apache also uses "The " elsewhere: https://repo.maven.apache.org/maven2/org/apache/apache/2/apache-2.pom.

¯\\\_(ツ)_/¯

**Pull request merged**